netbox/tenancy/views.py

```diff
@@ -7,17 +7,35 @@
 from ipam.models import Aggregate, ASN, IPAddress, IPRange, L2VPN, Prefix, VLAN, VRF
 from netbox.views import generic
 from utilities.utils import count_related
-from utilities.views import register_model_view
+from utilities.views import register_model_view, ViewTab
 from virtualization.models import VirtualMachine, Cluster
 from wireless.models import WirelessLAN, WirelessLink
 from . import filtersets, forms, tables
 from .models import *
 
 
+class ChildContactView(generic.ObjectChildrenView):
```

Comment:
We should rename this to `ObjectContactsView` to be consistent with similar view classes.